pdc_client/__init__.py

```diff
@@ -125,7 +125,10 @@ class to perform requests.
             if insecure is not None:
                 sys.stderr.write("Warning: '%s' option is deprecated; please use '%s' instead\n" % (
                     CONFIG_INSECURE_KEY_NAME, CONFIG_SSL_VERIFY_KEY_NAME))
-                ssl_verify = not insecure
+                if insecure:
+                    ssl_verify = not insecure
+                else:
+                    ssl_verify = ssl_verify or not insecure
```

Comment:
I am doubting if ssl_verify can get path from the config file(Line 123) or not. As code in  pdc_client/runner.py#L154(ssl_verify = self.args.ca_cert or not self.args.insecure)
ssl_verify is path/True/False, it will not be None, right? Please correct me, maybe i misunderstand something